### PR TITLE
CPF-103 Update User model

### PR DESCRIPTION
**Pull request by Vikariusu**
# Ticket #103 

## Description
Updated the `User` model to have all fields required and removed `validated`, `invalidated` and `subrecipients` relations. Tests have been also updated to include all required fields and match changes in the schema.

## Warning
**Changes to the schema break some existing functionality, mainly fetching and creating/updating users.**

### CODE REVIEW

File: api/db/schema.prisma [hunk 1]
```diff
@@ -37,19 +36,15 @@ model Organization {
 model User {
   id             Int                 @id @default(autoincrement())
   email          String
```
Comment (Vikariusu) [inline api/db/schema.prisma:38]: Question: Do we want to implement email validation when calling create/update user or is there a way to enforce it here? Also, do we want to use `unique`?

Comment (as1729) [reply api/db/schema.prisma:38]: yes, we would like to use unique here as well 👍 =1

Comment (Vikariusu) [reply api/db/schema.prisma]: done ✅

**Review by as1729 — APPROVED** [review]
Looks good!

**Pull request merged**